chapters/annotations.tex

```diff
@@ -1349,18 +1347,18 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
   of a (\lstinline!connectorSizing!) parameter. \emph{Example:} Assume there is a
   connector \lstinline!inPorts! and a component \lstinline!step1!:
 \begin{lstlisting}[language=modelica]
-  parameter Integer nIn=0 annotation(Dialog(connectorSizing=true));
-  StepIn inPorts[nIn];
-  Step step1(nIn=0);
+parameter Integer nIn=0 annotation(Dialog(connectorSizing=true));
+StepIn inPorts[nIn];
+Step step1(nIn=0);
 \end{lstlisting}
   Drawing a connection line between connectors \lstinline!inPorts! and
   \lstinline!step1.inPorts! results in:
 \begin{lstlisting}[language=modelica]
-  parameter Integer nIn=0 annotation(Dialog(connectorSizing=true));
-  StepIn inPorts[nIn];
-  Step step1(nIn=nIn); // nIn=0 changed to nIn=nIn
-  equation
-  connect(inPorts, step1.inPorts); // new connect equation
+parameter Integer nIn=0 annotation(Dialog(connectorSizing=true));
```

Comment:
I believe this code-fragment should be formatted differently - so that only "equation" is un-indented.

Reply:
Fixed in 7e78238e090f0288b3978adfabbc6cf372eba3cd.